src/encoded/types/antibody_lot.py

```diff
@@ -158,151 +162,166 @@ def lot_reviews(characterizations, targets, request):
 
-    status_ranking = {
-        'characterized to standards': 10,
-        'characterized to standards with exemption': 9,
-        'compliant': 8,
-        'exempt from standards': 7,
-        'pending dcc review': 6,
-        'partially characterized': 5,
-        'awaiting characterization': 4,
-        'not characterized to standards': 3,
-        'not pursued': 2,
-        'not compliant': 1,
-        'not reviewed': 0,
-        'not submitted for review by lab': 0,
-        'deleted': 0,
-        'in progress': 0
-    }
+        # Get the highest ranking status in the set
+        secondary_statuses.sort(key=lambda x: status_ranking[x], reverse=True)
+        secondary_status = secondary_statuses[0]
 
-    if secondary_chars:
-        secondary_status = get_secondary_status(secondary_chars, status_ranking)
+    # If there are no primaries, return the lot review with the secondary status
+    if not primary_chars:
+        # The default if no primary characterizations have been submitted
+        base_review['status'] = ab_states[(None, secondary_status)]
+        base_review['detail'] = ab_state_details[(None, secondary_status)]
+        if base_review['status'] == 'not pursued':
+            base_review['biosample_term_name'] = 'at least one cell type or tissue'
+            base_review['biosample_term_id'] = 'NTR:00000000'
+        return [base_review]
 
-    # Now check the primaries and update their status accordingly
+    # Done with easy cases, the remaining require reviews.
+    # Check the primaries and update their status accordingly
     lot_reviews = build_lot_reviews(primary_chars,
                                     secondary_status,
                                     status_ranking,
-                                    base_review,
                                     review_targets,
                                     is_histone_mod,
-                                    target_organisms)
+                                    char_organisms)
 
     return lot_reviews
 
 
-def get_secondary_status(secondary_chars, status_ranking):
-    # Determine the consensus secondary characterization status based on
-    # all those submitted if more than one
-    secondary_statuses = [item['status'] for item in secondary_chars]
-
-    # Get the highest ranking status in the set
-    secondary_statuses.sort(key=lambda x: status_ranking[x], reverse=True)
-
-    return secondary_statuses[0]
-
-
 def build_lot_reviews(primary_chars,
                       secondary_status,
                       status_ranking,
-                      base_review,
                       review_targets,
                       is_histone_mod,
-                      target_organisms):
+                      char_organisms):
+
+    # We have primary characterizatons
+    char_reviews = {}
+    lane_organism = None
+    key = None
+    for primary in primary_chars:
+        # The default base review
+        base_review = {
```

Comment:
It confuses me that this is named the same as the base_review (default no characterizations) in line 67.